slaves/linux-cross/Dockerfile

```diff
@@ -101,10 +103,17 @@ USER root
 # Rename all the compilers we just built into /usr/bin and also without
 # `-unknown-` in the name because it appears lots of other compilers in Ubuntu
 # don't have this name in the component by default either.
+# Also the aarch64 compiler is prefixed with `aarch64-unknown-linux-gnueabi`
+# by crosstool-ng, but Ubuntu just prefixes it with `aarch64-linux-gnu` so
+# we'll, additionally, strip the eabi part from its binaries.
 RUN                                                                           \
   for f in `ls /x-tools/*-unknown-linux-*/bin/*-unknown-linux-*`; do          \
     g=`basename $f`;                                                          \
     ln -vs $f /usr/bin/`echo $g | sed -e 's/-unknown//'`;                     \
+  done && \
+  for f in `ls /usr/bin/aarch64-linux-gnueabi-*`; do                          \
```

Comment:
The first `for` created these symlinks:

```
/usr/bin/aarch64-linux-gnueabi-gcc -> /x-tools/aarch64-unknown-linux-gnueabi/bin/aarch64-unknown-linux-gnueabi-gcc
```

The second `for` renames these links to remove the `eabi` part.